src/utils/daos_autotest.c

```diff
@@ -64,13 +64,13 @@ step_print(const char *status, const char *comment, va_list ap)
 	int	i;
 
 	end = clock();
-	printf("  %s    ", status);
+	fprintf(autotest_ap->outstream, "  %s    ", status);
 	sprintf(timing, "%03.3f", duration());
 	for (i = strlen(timing); i < 7; i++)
 		printf(" ");
```

Comment:
This should be using autotest_ap->outstream as well presumably?

Reply:
Yes, one of several instances I missed in the constant game of whack-a-mole I was playing as master shifted underneath my PR.